src/components/LoginForm/LoginForm.tsx

```diff
@@ -19,6 +19,14 @@ import { addQueryParam } from '#src/utils/location';
 import type { FormErrors } from '#types/form';
 import type { LoginFormData } from '#types/account';
 
+const LOGIN_FORM_MESSAGES = ['simultaneous_logins'] as const;
+
+export type LoginFormMessage = (typeof LOGIN_FORM_MESSAGES)[number];
+
+const isValidLoginFormMessage = (message: string | undefined): message is LoginFormMessage => LOGIN_FORM_MESSAGES.includes(message as LoginFormMessage);
+
+export const getLoginFormMessage = (message: string | undefined): LoginFormMessage | undefined => (isValidLoginFormMessage(message) ? message : undefined);
+
```

Comment:
@dbudzins @borkopetrovicc did you perhaps agree on this outside the review comments? I thought the conclusion was to not cast the message to `LoginFormMessage` but just pass it as a string.

Reply:
No, we didn't consult outside of here. I tried to make my solution better by making it as type safe as possible.

Reply:
This is a lot of boilerplate that we don't really need and is actually making the components less scalable imo. Strict type enforcement is not always the best choice. Can you please give a go at the solution suggested in this conversation: https://github.com/jwplayer/ott-web-app/pull/321#discussion_r1260197777

Reply:
967073ed87863665ef7350fe545aba8b5be69efb